robocop/checkers/spacing.py

```diff
@@ -472,3 +472,23 @@ def get_indent(node):
                 break
             indent_len += len(token.value.expandtabs(4))
         return indent_len
+
+
+class LeftAlignedChecker(VisitorChecker):
+    """ Checker for left align. """
+    rules = {
+        "1014": (
+            "variable-should-left-aligned",
+            "Variable in variable section should be left aligned.",
+            RuleSeverity.ERROR
+        )
+    }
+
+    def visit_VariableSection(self, node):  # noqa
+        for child in node.body:
+            if not child.data_tokens:
+                continue
+            tokens = child.data_tokens
+            if tokens[0].type == Token.VARIABLE and tokens[0].value == '':
```

Comment:
It looks like it will always trigger even for correctly aligned - because first token will be non data separator. It also doesnt work for RF 3.2 for some reason - check cli results in github